code/datums/gamemodes/_game_mode.dm

```diff
@@ -122,6 +122,8 @@ GLOBAL_VAR(common_report) //Contains common part of roundend report
 ///Gamemode setup run after the game has started
 /datum/game_mode/proc/post_setup()
 	addtimer(CALLBACK(src, PROC_REF(display_roundstart_logout_report)), ROUNDSTART_LOGOUT_REPORT_TIME)
+	addtimer(CALLBACK(GLOBAL_PROC, GLOBAL_PROC_REF(send_global_signal), COMSIG_GLOB_DEPLOY_TIMELOCK_ENDED), deploy_time_lock)
```

Comment:
this triggers regardless of gamemode? Seems smelly. Should be on infestation or whatever, although custom squads also seems wew on crash, so maybe nukewar specifically.